compiler-rt/test/asan/TestCases/use-after-free-symbolizer-markup.cpp

```diff
@@ -0,0 +1,33 @@
+// COM: End to end test for the sanitizer symbolizer markup. Since it uses debug info
+// COM: to do offline symbolization we only check that the current module is correctly symbolized
+// REQUIRES: linux
```

Comment:
Is this intended to change? If so, a comment/TODO may be appropriate.

Reply:
I don't think it is intended to change, at least no for now.